shared/oae/css/oae.components.css

```diff
@@ -198,6 +194,86 @@ i.fa.large:before {
 }
 
 
+/****************
+ ** SWITCH TAB **
+ ****************/
+
+.oae-switchtab-container {
+    text-align: center;
+}
+
+.oae-switchtab-container .oae-switchtab-control {
+    background-color: #333;
+    border: solid 3px #333;
+    display: inline-block;
+    font-size: 15px;
+}
+
+.oae-switchtab-container .oae-switchtab-control .oae-switchtab-switch {
+    background: transparent;
+    border: 0;
+    color: #DDD;
+    min-width: 170px;
+    padding: 8px 0;
+}
+
+.oae-switchtab-container .oae-switchtab-control,
+.oae-switchtab-container .oae-switchtab-control .oae-switchtab-switch {
+    border-radius: 3px;
+}
+
+.oae-switchtab-container .oae-switchtab-control .oae-switchtab-switch > span {
+    color: inherit;
+    position: relative;
+}
+
+/* Give the first (left-most) element space on the left side to make the button bigger */
+.oae-switchtab-container .oae-switchtab-control .oae-switchtab-switch > span:first-child {
+    margin-left: 35px;
+}
+
+/* Give the last (right-most) element space on the right side to make the button bigger */
+.oae-switchtab-container .oae-switchtab-control .oae-switchtab-switch > span:last-child {
+    margin-right: 35px;
+}
+
+/* Icon generally takes up the most vertical baseline space, so shift it down to help align it with
+   the label content */
+.oae-switchtab-container .oae-switchtab-control .oae-switchtab-switch-icon,
+.oae-switchtab-container .oae-switchtab-control .oae-switchtab-switch-icon:hover {
+    top: 1px;
+}
+
+/* Shift the text up one pixel to help align it to larger button. Also give it some space on the
+   left */
+.oae-switchtab-container .oae-switchtab-control .oae-switchtab-switch-label {
+    bottom: 1px;
+    margin-left: 5px;
+}
+
+/* Highlight the active "switch" by giving it a background */
+.oae-switchtab-container .oae-switchtab-control .oae-switchtab-switch.oae-switchtab-active {
+    background-color: #EEE;
+    color: #333;
+}
+
+/* Highlight the active "switch"'s icon with a color, how nice! */
+.oae-switchtab-container .oae-switchtab-control .oae-switchtab-switch.oae-switchtab-active .oae-switchtab-switch-icon {
+    color: #3481BE;
```

Comment:
I'm not exactly sure what skin variable this would be congruent with, but my assumption is that this colour should actually be brand-able.